code/datums/religions.dm

```diff
@@ -125,6 +128,12 @@
 		preacher.put_in_hands(holy_book)
 	religiousLeader = preacher.mind
 	convert(preacher, null)
+	if(keys[1]=="christianity")
```

Comment:
Surely there's a better way.

Reply:
Fielding suggestions. Here are the main routes I thought of:
1) Check for the exact type path of /datum/religion, which felt bad
2) Give default religion a var called mass_convert set to TRUE and set every child to FALSE. This sucks mostly for line bloat
3) Check the religion name. This is not terrible but you could name a custom religion Christianity I guess?
4) Do this and check the key

Reply:
Could just have christianity be a subtype of the default religion datum and do it OO

Reply:
I think it was the intent of shifty that the top religion be the default and therefore not an abstract

You are right though, we could do it that way